cpp/examples/tpch/utils.hpp

```diff
@@ -0,0 +1,454 @@
+/*
+ * Copyright (c) 2024, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <iostream>
+#include <chrono>
+#include <ctime>
```

Comment:
Please make sure clang-format is running on these. We typically put [cudf includes](https://github.com/rapidsai/cudf/blob/39de5a2527b297ba79c625993a49b28c3baf5b00/.clang-format#L82) before [STL includes](https://github.com/rapidsai/cudf/blob/39de5a2527b297ba79c625993a49b28c3baf5b00/.clang-format#L96).